v3/src/components/graph/models/axis-model.ts

```diff
@@ -12,9 +12,27 @@ export const attrPlaceToAxisPlace: Partial<Record<GraphAttrPlace, AxisPlace>> =
   topSplit: "top"
 }
 
+export const axisPlaceToGraphAttrPlace = (place: AxisPlace) => {
+  let attrPlace:GraphAttrPlace
+  switch (place) {
+    case 'bottom':
+      attrPlace = 'x'
+      break
+    case 'left':
+      attrPlace = 'y'
+      break
+    case 'top':
+      attrPlace = 'topSplit'
+      break
+    case 'right':
+      attrPlace = 'y2'  // Todo: how to deal with 'rightSplit'?
+  }
+  return attrPlace
+}
```

Comment:
Seems like you could/should follow the same pattern as `attrPlaceToAxisPlace` for consistency (and brevity):
```suggestion
export const axisPlaceToAttrPlace: Record<AxisPlace, GraphAttrPlace> = {
  bottom: "x",
  left: "y",
  top: "topSplit",
  right: "y2"  // Todo: how to deal with 'rightSplit'?
}
```